doc/plutus-core-spec/cardano/versions.tex

```diff
@@ -106,6 +106,7 @@ \section{Ledger languages}
         \LL{PlutusV2} & 7.0 & Batch 2 \\
         \LL{PlutusV2} & 8.0 & Batch 3 \\
         \LL{PlutusV3} & 9.0 & Batch 4 \\
+        \LL{??} & ?? & Batch X \\
```

Comment:
This table will need to be reworked or discarded. Since we can now add (new or existing) builtins to existing Plutus versions, the table may no longer make sense. This particular builtin will likely be introduced into V1, V2 and V3 at the same time, at Protocol version 11.0.

Reply:
> This table will need to be reworked or discarded. Since we can now add (new or existing) builtins to existing Plutus versions, the table may no longer make sense. This particular builtin will likely be introduced into V1, V2 and V3 at the same time, at Protocol version 11.0.

I think there's an issue to fix that.  I'll see if I can fid it.